nautobot/docs/installation/region-and-site-data-migration-guide.md

```diff
@@ -0,0 +1,405 @@
+# Region and Site to Location Migration Guide
+
+In Nautobot 2.0.0, all the `Region` and `Site` related data models are being migrated to use `Location`. Below is a comprehensive guide for Nautobot users and Nautobot App developers to migrate their `Region` and `Site` related models and apps to `Location`.
+
+## Migrate ObjectPermission instances in Nautobot from Region and Site to Location
+
+Nautobot admins need to migrate `Site` and `Reigon` Related `ObjectPermission` instances to `Location`. The correct way to do it are documented below with practical examples.
+
+### Region Specific ObjectPermission
+
+Add `Location` ContentType to `object_types` field of the `ObjectPermission` if it is not already present.
```

Comment:
Here and below, is "add" correct, or should the user "replace" the Region `object_types` entry with a Location entry?